includes/templates/template_default/templates/tpl_modules_categories_tabs.php

```diff
@@ -4,22 +4,32 @@
  *
  * Template stub used to display categories-tabs output
  *
- * @copyright Copyright 2003-2022 Zen Cart Development Team
+ * @copyright Copyright 2003-2024 Zen Cart Development Team
  * @copyright Portions Copyright 2003 osCommerce
  * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
- * @version $Id: DrByte 2020 Jul 10 Modified in v1.5.8-alpha $
+ * @version $Id: v2.1.0-alpha3 $
  */
 
-  include(DIR_WS_MODULES . zen_get_module_directory(FILENAME_CATEGORIES_TABS));
+include(DIR_WS_MODULES . zen_get_module_directory(FILENAME_CATEGORIES_TABS));
```

Comment:
Why `include` instead of `require`?  In either case, the wrapping parentheses aren't needed.